bin/bank/pycbc_brute_bank

```diff
@@ -278,29 +312,54 @@ def draw(rtype):
     params = {}
 
     if rtype == 'uniform':
-        for name, pmin, pmax in zip(args.params, args.min, args.max):
-            params[name] = numpy.random.uniform(pmin, pmax, size=size)
+        if args.input_config is None:
+            for name, pmin, pmax in zip(args.params, args.min, args.max):
+                params[name] = numpy.random.uniform(pmin, pmax, size=size)
+        else:
+            # `draw_samples_from_config` has its own fixed seed, so must overwrite it.
+            random_seed = numpy.random.randint(low=0, high=2**32-1)
+            samples = draw_samples_from_config(args.input_config, size, random_seed)
+            for name in samples.fieldnames:
+                params[name] = samples[name]
+
     elif rtype == 'kde':
         trail = 300
         if trail > len(bank):
             trail = len(bank)
         p = bank.keys()
         p = [k for k in p if k not in fdict]
         p.remove('approximant')
+        if args.input_config is not None and 'q' in static_args:
```

Comment:
Let's avoid hardcoding any parameters. I think what we want is if you provide a config file, the kde should be drawn in the variable parameters you choose (only) and then you make you add in the static args and apply the same conversions / transforms you get from the config file. That way it is completely generic, make sense?

Reply:
This also avoids passing anything to the kde that is singular since the variable params by definition *must* vary.

Reply:
This also avoids passing anything to the kde that is singular since the variable params by definition *must* vary.